tests/test_wave_01.py

```diff
@@ -158,13 +149,10 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert updated_data["watched"]
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
+    assert not updated_data["watchlist"]
```

Comment:
A more explicit `assert` would be:
```suggestion
    assert updated_data["watchlist"] == []
```